src/main/groovy/com/palantir/gradle/javadist/JavaDistributionPlugin.groovy

```diff
@@ -58,19 +59,37 @@ class JavaDistributionPlugin implements Plugin<Project> {
             }
         }
 
+        Task copyLauncherBinaries = project.tasks.create('copyLauncherBinaries', {
+            group = GROUP_NAME
+            description = "Creates go-java-launcher binaries."
+        }) << {
+            // TODO(rfink) Don't check these in, but fetch them from a published artifact?
+            ["javalauncher-linux-amd64", "javalauncher-darwin-amd64"].each { file ->
+                def dest = Paths.get("${project.buildDir}/scripts/${file}")
```

Comment:
This should probably end up in var/conf. config.sh was in the immutable section because it was a script (and thus vulnerable to tampering).